ntbs-ui-tests/Steps/Steps.cs

```diff
@@ -43,19 +43,39 @@ public void GivenIAmOnPage(string pageName)
         [When(@"I enter (.*) into '(.*)'")]
         public void WhenIEnterValueIntoFieldWithId(string value, string elementId)
         {
+            FindById(elementId).Click();
             FindById(elementId).SendKeys(value);
         }
 
+        [When(@"I wait")]
+        public void WhenIWait()
+        {
+            Thread.Sleep(2000);
+        }
+
+        [When(@"I click away from element '(.*)'")]
+        public void WhenITabOutOfElement(string elementId)
+        {
+            FindById(elementId).SendKeys("\t");
+            Thread.Sleep(2000);
+        }
+
         private IWebElement FindById(string elementId)
         {
             return Browser.FindElement(By.Id(elementId));
         }
 
         [When(@"I check '(.*)'")]
         [When(@"I select radio value '(.*)'")]
-        [When(@"I click on '(.*)'")]
+        public void WhenISelectRadioOrCheckbox(string elementId)
+        {
+            FindById(elementId).Click();
+        }
+
+        [When(@"I click on the '(.*)' button")]
         public void WhenIClickOn(string elementId)
         {
+            Thread.Sleep(1000);

```

Comment:
How come those need a wait?

Reply:
Turns out they don't - this was from when I was playing around with it earlier